### ISSUE TITLE
I thought the fee for market orders on Binance was 0.5%, not 0.05% ?

### ISSUE THREAD

**Issue by gabriel-lima-quickbit** [whittlem/pycryptobot#42]
https://github.com/whittlem/pycryptobot/blob/ae88713ca587bd4a06b85a40df76779362b75a16/models/Binance.py#L84

I believe the Binance fee is 0.1% for limit orders, but 0.5% on for market orders. 
So I think the row on line 84 should be:
base_quantity = base_quantity * 0.995
rather than 
base_quantity = base_quantity * 0.9995
right?

**Comment by whittlem** [comment]
I'm not entirely sure. I use Coinbase Pro myself. The fee structure there seems a little more straight forward. I've just been looking at, "https://www.binance.com/en/fee/schedule". It looks like the fees can vary a lot depending on conditions. If you are sure about the update above I can change it. If anyone else could confirm that would be helpful?

**Comment by gabriel-lima-quickbit** [comment]
I know, it is not very clear. I've seen this:
https://www.binance.us/en/fee/schedule
"General: 0.1% spot trading fee; 0.5% Instant Buy/Sell fee."

And then on this page:
https://academy.binance.com/en/articles/what-is-a-market-order
"Unlike limit orders, where orders are placed on the order book, market orders are executed instantly at the current market price, meaning that you pay the fees as a market taker."

But it is not very clear at all. So I agree, if someone else could shed some light on this, it would be much appreciated.

**Comment by gabriel-lima-quickbit** [comment]
The only thing I found was this:
https://leapfrogs.navy/how-to-lower-your-binance-trading-fees/

Which also mentions the spot market order fee being 0.5%.

**Comment by whittlem** [comment]
Are you sure that base quantity needs to change? A lot of people are using the Binance bot and they aren't reporting any issues.

**Comment by gabriel-lima-quickbit** [comment]
No need to change it on my behalf. I am not using the bot at all :-)
I just noticed this because I was lurking around your code, as I am toying with writing a bot on my own, and wanted to see if you had implemented some way to create limit orders, rather than market orders.
Which took me to that specific piece in your code, where I noticed said value. So just keep it. But if someone complains, then you know where to look. :-)
